README.md

```diff
@@ -102,9 +106,23 @@ These are the supported vector properties:
 
 #### points
 
-The `points` setting defines a list of points to be drawn on the
-board. It is optional. Each point should be defined as an object with
-`name` and `coords` properties.
+The `points` setting defines a list of points to be either just drawn
+on the board for reference, or to be placed by the student.  Points can
+have the following properties:
+
+* `name` (required): The unique name of the point, used in checks and
+  internally to identify the point.
+* `coords` (required): An array of length 2 with the (initial)
+  coordinates of the point.
+* `fixed`: Whether the point should be drawn in a fixed location or
+  placed by the student.  Defaults to `true`.
+* `render`: Whether the point should be rendered when the board is
```

Comment:
@smarnach "render" is more of a technical term, while this readme is mostly user-oriented. Consider changing to "shown"